prespective_rectification/abrupt_corners.py

```diff
@@ -0,0 +1,131 @@
+import imutils
+import cv2
+import matplotlib.pyplot as plt
+import pandas as pd
+import numpy as np
+from scipy.signal import savgol_filter
+
+from prespective_rectification.ploting import plot_points
+
+
+def select_points_by_segment(point_start, point_end, len_points):
```

Comment:
Would be nice to add a short docstring for each method with a short description of each parameter. Point start/end are not very clear for me atm.

Reply:
sure!